src/components/common/chat-popup.vue

```diff
@@ -25,22 +25,31 @@
 
       <v-tooltip top content-class="top-tooltip" nudge-top="5">
         <template v-slot:activator="{ on, attrs }">
-          <v-btn
-            :disabled="(chatStatus !== 'open')"
-            large
-            outlined
-            color="bcgovblue"
-            elevation="4"
-            class="chat-button"
-            type="submit"
-            v-bind="attrs"
-            v-on="on"
-            aria-label="Chat with Helpdesk staff">
-            <v-icon class="mr-2 ml-n2">mdi-forum-outline</v-icon>
-            <span class="font-weight-bold">Chat</span>
-          </v-btn>
+          <div id="chat-button-wrapper" v-on="on" v-bind="attrs">
```

Comment:
Since you have the v-on on this div now, should it be removed from the template on line 27?

Reply:
No, we need to keep it to "repass" it on to v-btn.